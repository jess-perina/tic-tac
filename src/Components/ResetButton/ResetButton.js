import React from 'react'

function ResetButton(props) {
  return (
    <button
      type="button"
      className="reset-btn btn waves-effect waves-light" onClick={() => props.resetGame()}>Reset Game
    </button>
  )
}

export default ResetButton
