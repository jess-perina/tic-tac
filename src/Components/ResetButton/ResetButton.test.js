import React from 'react'
import { shallow } from 'enzyme'
import ResetButton from './ResetButton'

describe('ResetButton Component', () => {
  it('renders as espected', () => {
    expect(shallow(<ResetButton />)).toMatchSnapshot()
  })
})