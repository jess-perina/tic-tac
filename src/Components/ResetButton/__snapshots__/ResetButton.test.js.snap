// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetButton Component renders as espected 1`] = `ShallowWrapper {}`;
